openmdao/core/problem.py

```diff
@@ -70,6 +72,8 @@ class Problem(object):
         2 -- The `final_setup` has been run, everything ready to run.
     """
 
+    _post_setup_func = None
```

Comment:
Is this going to hurt nested problems if you want different post setup hook functions in each?

Reply:
There are always things we can do if we choose to support that sort of use case, but I think it'll be very uncommon and we shouldn't make this more complicated unless users actually request that sort of thing.

Reply:
Ok, but I should note that I was helping John get a nested problem to work right before Thanksgiving.

Reply:
nested problems??? for what application?

Reply:
I used nested problems to enable a bracketed root finding algorithm with a composite residual function

Reply:
like the brent method?

Reply:
yes